### ISSUE TITLE
Clearer naming

### ISSUE THREAD

**Issue by jonathan-dejong** [khromov/acf-local-json-manager#1]
Yo! 

Probably gonna end up making lots of requests and suggestions as time goes by with this project I'm working on.. But what strikes me almost immediately is the naming of "Disable overrides". 

It sounds overly harsh and scary for what it does. Generella what it does is just allow ACF to get stored and loaded from the active theme as per it's default functionality. I think it could be much clearer (and less scary). Perhaps something like picking up the theme name with an appended (default behavior):
`Twentysixteen (default behavior)`
 for example.

**Comment by khromov** [comment]
@jonathan-dejong I agree with the label, but we wouldn't be able to have `Twentysixteen (default behavior)` because we don't know that the theme has the final say (it depends on load order and hook priority).

In fact, we can't even resolve the default path and show it as a tooltip because we use the same hook (  `acf/settings/save_json` ) to override the behaviour...

Maybe simply replace the current text with `Default behaviour` ?

**Comment by jonathan-dejong** [comment]
@khromov Yeah I suppose that'd work. I didn't think of that.. However what would qualify as the default behavior then? When I set "disable overrides" what do you do? Remove all plugins where we have the save_json hook AND this plugins hook?

**Comment by khromov** [comment]
@jonathan-dejong Setting "disable overrides" only removes the override hook for this plugin ( https://github.com/khromov/acf-local-json-manager/blob/master/acf-local-json-manager.php#L21 ) , so the result will be a state as-if the ACF Local JSON Manager was not installed.

**Comment by jonathan-dejong** [comment]
Ahhh okay I see :)
